config/_default/params.toml

```diff
@@ -12,9 +12,9 @@ mainSections = ["docs"]
 # Doks (@thulite/doks-core)
 [doks]
   # Color mode
-  # For now, all PNG text is White so switching to Light mode will make those texts disappear.
-  colorMode = "dark" # auto (default), light or dark
-  colorModeToggler = false # true (default) or false (this setting is only relevant when colorMode = auto)
+  # For now, all PNG text is White so switching to Light mode will make those texts disappear
+  colorMode = "dark"            # Use "auto", "light", or "dark"
```

Comment:
why is this not on auto actually?

Reply:
See Piaras comment: # For now, all PNG text is White so switching to Light mode will make those texts disappear. I kept this untouched and put it onto the to-do list for the next page version. We need to have Images/Icons that match the chosen mode. I checked the light mode during my tests and the page looks awful with it.